prov/efa/src/efa.h

```diff
@@ -122,6 +122,11 @@
 /* maximum name length for shm endpoint */
 #define EFA_SHM_NAME_MAX	   (256)
 
+#define EFA_DEF_RUNT_SIZE (307200)
+#define EFA_DEF_INTER_MAX_MEDIUM_MESSAGE_SIZE (65536)
+#define EFA_DEF_INTER_MIN_READ_MESSAGE_SIZE (1048576)
+#define EFA_DEF_INTER_MIN_READ_WRITE_SIZE (65536)
+
```

Comment:
should these include DEFAULT in the name, since it looks like they could be overwritten based on fi_param_get ?

Reply:
I'm following the convention set elsewhere in `efa.h` (ex: `EFA_DEF_POOL_ALIGNMENT`, `EFA_DEF_CQ_SIZE`). I infer `DEF` to mean "default" here, but it could simply be "definition."

Reply:
I think change to `DEFAULT` is better.  we should change the old ones.

Reply:
I can easily `s/EFA_DEF_/EFA_DEFAULT_/g`, but it would touch multiple files (only for `EFA_DEF_CQ_SIZE` since `EFA_DEF_POOL_ALIGNMENT` is actually unused). Do you want that as part of this PR?

Reply:
No. Do not change the old ones in this PR. Just start using `DEFAULT` for new ones in this PR.

Reply:
Ack, see latest: 52ee1f426a27db5a4a3276b748151bfa7ddaf048..1d3f6842f5912baa4e926e6e08b914af2d671621